data-pipeline/src/pipeline/input_schemas.py

```diff
@@ -262,146 +262,122 @@
-    "Receipts from supply teacher insurance claims": "float",
-    "Donations and/or voluntary funds": "float",
-    "Other self-generated income": "float",
-    "Investment income": "float",
-    "Teaching staff": "float",
-    "Supply teaching staff": "float",
-    "Education support staff": "float",
-    "Administrative and clerical staff": "float",
-    "Premises staff": "float",
-    "Catering staff": "float",
-    "Other staff": "float",
-    "Indirect employee expenses": "float",
-    "Staff development and training": "float",
-    "Staff-related insurance": "float",
-    "Supply teacher insurance": "float",
-    "Building and Grounds maintenance and improvement": "float",
-    "Cleaning and caretaking": "float",
-    "Water and sewerage": "float",
-    "Energy": "float",
-    "Rent and Rates": "float",
-    "Other occupation costs": "float",
-    "Special facilities": "float",
-    "Learning resources (not ICT equipment)": "float",
-    "ICT learning resources": "float",
-    "Examination fees": "float",
-    "Educational Consultancy": "float",
-    "Administrative supplies - non educational": "float",
-    "Agency supply teaching staff": "float",
-    "Catering supplies": "float",
-    "Other insurance premiums": "float",
-    "Legal & Professional": "float",
-    "Auditor costs": "float",
-    "Interest charges for Loan and Bank": "float",
-    "Direct revenue financing (Revenue contributions to capital)": "float",
-    "PFI Charges": "float",
-    "Revenue Reserve": "float",
-    "Grant Funding": "float",
-    "Direct Grant": "float",
-    "Community Grants": "float",
-    "Targeted Grants": "float",
-    "Self Generated Funding": "float",
-    "Total Income ": "float",
-    "Supply Staff": "float",
-    "Other Staff Costs": "float",
-    "Staff Total ": "float",
-    "Maintenance & Improvement": "float",
-    "Premises": "float",
-    "Catering Exp": "float",
-    "Occupation": "float",
-    "Supplies and Services": "float",
-    "Educational Supplies": "float",
-    "Brought in Professional Services": "float",
-    "Total Expenditure": "float",
+    "Lead_UPIN": "Int64",
+    "BNCH11110T (EFA Revenue Grants)": "float",
+    "BNCH11131 (DfE Family Revenue Grants)": "float",
+    "BNCH11141 (SEN)": "float",
+    "BNCH11142 (Other Revenue)": "float",
+    "BNCH11151 (Other Government Revenue Grants)": "float",
+    "BNCH11161 (Government source (non-grant))": "float",
+    "BNCH11162 (Academies)": "float",
+    "BNCH11163 (Non- Government)": "float",
+    "BNCH11123-BTI011-A (MAT Central services - Income)": "float",
+    "BNCH11201 (Income from facilities and services)": "float",
+    "BNCH11202 (Income from catering)": "float",
+    "BNCH11203 (Receipts from supply teacher insurance claims)": "float",
+    "BNCH11300T (Voluntary income)": "float",
+    "BNCH11204 (Other income - revenue)": "float",
+    "BNCH11205 (Other Income from facilities and services)": "float",
+    "BNCH11400T (Investment income)": "float",
```

Comment:
Any difference between this and `BNCH11401 (Investment income)`?

Reply:
Don't know what the T prefix stands for. I was just refering to the cost mapping sheet. This is one for Alan and Assad